gui/mozregui/skip_chooser.py

```diff
@@ -17,9 +17,9 @@ def __init__(self, future_build_info, x=0, y=0, selectable=True):
         QGraphicsRectItem.__init__(self, x, y, self.WIDTH, self.WIDTH)
         self.future_build_info = future_build_info
         if selectable:
-            self.setFlags(self.ItemIsSelectable | self.ItemIsFocusable)
+            self.setFlags(BuildItem.ItemIsSelectable | BuildItem.ItemIsFocusable)
```

Comment:
I don't understand where `self` -> `BuildItem`, could you explain briefly?

Reply:
So in the upgraded version of PySide6, these are class attributes and are not accessible on their respective instances. Here's an example of what happens:

```
Traceback (most recent call last):
  File "/home/runner/work/mozregression/mozregression/gui/mozregui/log_report.py", line 60, in on_log_received
    cursor_to_add.movePosition(cursor_to_add.End)
AttributeError: 'PySide6.QtGui.QTextCursor' object has no attribute 'End'
```

I couldn't find anything documented for this, however these attributes are actually proxies to other class attributes (e.g., `MoveOperation.End` or `GraphicsItemFlag.ItemIsFocusable`, etc...) and do not require the instance itself.

Reply:
thanks, I wanted to make sure this was intentional, even if it was hacking around to get to this point